src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -333,16 +330,18 @@ describe('ReactDOMServerIntegration', () => {
     });
 
     if (ReactDOMFeatureFlags.useFiber) {
-      itRenders('a array type children as a child', async render => {
-        class AppComponent extends React.Component {
```

Comment:
ok, but set  disableNewFiberFeatures=false:

` const ReactFeatureFlags = require('ReactFeatureFlags');`
` ReactFeatureFlags.disableNewFiberFeatures = false;`


It still have warning.